applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -15,1258 +15,1473 @@
 #define M_PI 3.1415926535897932384626433832795
 #endif // M_PI
 
-

```

Comment:
The formatting has been updated so now only the new changes appear for easier comparison. Same for the hpp too.